roxctl/central/export/deployments/deployments_test.go

```diff
@@ -0,0 +1,88 @@
+package deployments
+
+import (
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"github.com/spf13/cobra"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/fixtures"
+	pkgGRPC "github.com/stackrox/rox/pkg/grpc"
+	envMocks "github.com/stackrox/rox/roxctl/common/environment/mocks"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	ioMocks "github.com/stackrox/rox/roxctl/common/io/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+	"google.golang.org/grpc"
+)
+
+func TestExportDeployments(t *testing.T) {
+	fakeService := &fakeDeploymentService{tb: t}
+	conn, closeFunc, err := pkgGRPC.CreateTestGRPCStreamingService(
+		context.Background(),
+		t,
+		func(registrar grpc.ServiceRegistrar) {
+			v1.RegisterDeploymentServiceServer(registrar, fakeService)
+		},
+	)
+	require.NoError(t, err)
+	defer closeFunc()
+
+	mockCtrl := gomock.NewController(t)
+
+	var buf bytes.Buffer
+	ioMock := ioMocks.NewMockIO(mockCtrl)
+	ioMock.EXPECT().Out().Times(1).Return(&buf)
+
+	env := envMocks.NewMockEnvironment(mockCtrl)
+	env.EXPECT().GRPCConnection().Times(1).Return(conn, nil)
+	env.EXPECT().InputOutput().Times(1).Return(ioMock)
+
+	fakeCmd := &cobra.Command{}
+	flags.AddTimeoutWithDefault(fakeCmd, 10*time.Second)
+
+	cmd := Command(env)
+	err = cmd.RunE(fakeCmd, []string{})
+	assert.NoError(t, err)
+
+	var expectedDeploymentJSON = fixtures.GetExpectedJSONSerializedTestDeployment(t)
```

Comment:
This is used only here do we need to put it in a separated package? How about creating a json file and using go embed in test to load it?

Reply:
I tried to have a structured approach to serialization tests before I split the PR. Alerts were used in multiple places in the codebase and I created a fixture in order to not duplicate the test object and associated serialized state across all tests. When I then moved to other objects types, I tried to reproduce the pattern.
I can and will then follow your suggestion for objects which are used in only one file.